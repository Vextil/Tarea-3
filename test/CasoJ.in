# ColaPrioridad
InsertarCP brasil 8 uruguay 15 argentina 14 bolivia 1 colombia 1 peru 2 paraguay 2 FIN
ExtraerCP
ExtraerCP
ExtraerCP
ElementosCP 15 1
ImprimirLista 1
# Pila
Apilar a 3
Apilar b 5
Apilar d 4
Apilar c 3
Apilar m 2
Apilar p 1
Apilar r 6
Apilar s 10
Apilar d 10
Desapilar
Desapilar
Desapilar
Desapilar
Desapilar
# Procesos
AgregarProcesos chrome 28000 firefox 25400 ie 10000 FIN
EliminarProceso ie
ImprimirProcesos
ListarProcesos
MuyConsumidores 26000
Fin
