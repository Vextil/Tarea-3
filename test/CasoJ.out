1># ColaPrioridad
2>OK
3>bolivia - 1
4>colombia - 1
5>peru - 2
6>OK
7>Ordenada. uruguay 
8># Pila
9>OK
10>OK
11>OK
12>OK
13>OK
14>OK
15>OK
16>OK
17>OK
18>(10,d)
19>(10,s)
20>(6,r)
21>(1,p)
22>(2,m)
23># Procesos
24>Hay 4 procesos.
25>La memoria que se libera es 10000.
26>
(25400,firefox) 
(28000,chrome) 
(100,boot) 
27>boot chrome firefox 
28>chrome 
29>Fin
